src/types/kubernetes.ts

```diff
@@ -45,7 +46,10 @@ export class Network {
 
   public get valid(): boolean {
     const { name, ipRange } = this;
-    return name !== "" && ipRange !== "";
+    return name !== "" &&
+      ipRange !== "" &&
+      !!validateName(name) &&
```

Comment:
`validateName` returns `String` if its not valid
String turns into *true* boolean value
so !!true = true

so u get true for non valid value and this getter is about `valid` value
it should be !validteName & !validateIp
please clarify if I'm wrong